code/randomized_algorithms/src/README.md

```diff
@@ -1,4 +1,34 @@
-# cosmos
-Your personal library of every algorithm and data structure code that you will ever encounter
+# Randomized Algorithms
+A randomized algorithm is that which receives in addition to its input data,a stream of random bits for making random choices.
```

Comment:
A randomized algorithm is **an algorithm** which ...

Reply:
Sure. I'll make all suggested changes.
Thanks.!